Week1/homework/app.js

```diff
@@ -1,11 +1,157 @@
+    "meisje_in_de_trein",
+    "hanna's_verjaardag",
+    "Een_nieuwe_toekomst",
+    "de_geur_van_regen",
+    "detective_orhan",
+    "last_minute",
+    "de_man_van_de_bloemen",
+    "ren_amina_ren",
+    "het_diner"
   ];
 
+  //step 2.3
+//   function bookList () { 
+
+//     let bookcontainer = document.createElement("div");
+//     let ul = document.createElement("ul");
+//     bookcontainer.appendChild(ul);
+
+//     for (let i = 0; i< bookTitles.length; i++) {
+//       let li = document.createElement("li");
+//       ul.appendChild(li);
+//       let content = document.createTextNode(bookTitles[i]);
+//       li.appendChild(content);
+//     }
+//     console.log(bookcontainer);
+
+//   }
+// bookList();
+
+
+  //console.log(bookTitles);
 
   // Replace with your own code
-  console.log(bookTitles);
+  //console.log(bookTitles);
+
+
+  const bookDetails = {
+    "anna_frank_haar_leven": {
+    title: "Anna Frank haar leven",
+    language: "Dutch",
+    author: "Anna Frank",
+  },
+    "meisje_in_de_trein": {
+      title: "Meisje in de trein",
+      language: "Dutch",
+      author: "Paula Hawkins",
+    },
+    "hanna's_verjaardag": {
+      title: "Hanna's verjaardag",
+      language: "Dutch",
+      author: "Gull Akerblom",
+    },
+    "Een_nieuwe_toekomst": {
+      title: "Een nieuwe toekomst",
+      language: "Dutch",
+      author: "Conor Grennan",
+    },
+    "de_geur_van_regen": {
+      title: "De geur van regen",
+      language: "Dutch",
+      author: "Lieneke Dijkzeul",
+    },
+    "detective_orhan": {
+      title: "Detective Orhan",
+      language: "Dutch",
+      author: "Sadik Yemni",
+    },
+    "last_minute": {
+      title: "Last Minute",
+      language: "Dutch",
+      author: "Linda van Rijn",
+    },
+    "de_man_van_de_bloemen": {
+      title: "De man van de bloemen",
+      language: "Dutch",
+      author: "Johan van Caeneghem",
+    },
+    "ren_amina_ren": {
+      title: "Ren Amina ren",
+      language: "Dutch",
+      author: "Annelie Drewsen",
+    },
+    "het_diner": {
+      title: "Het diner",
+      language: "Dutch",
+      author: "Herman Koch",
+    }
+  
+  }
+  //console.log(bookDetails);
+  
+  const bookImg = {
+    anna_frank_haar_leven : "images/Anna Frank.jpeg",
+    meisje_in_de_trein : "images/meisje in de trein.jpg",
+    hannas_verjaardag : "images/Hannas' verjaardag.jpg",
+    Een_nieuwe_toekomst : "images/nieuwe toekomst.jpg",
+    de_geur_van_regen : "images/de geur.jpg",
+    detective_orhan : "images/detective orhan.jpg",
+    last_minute : "images/last minute.jpg",
+    de_man_van_de_bloemen : "images/man van de bloemen.jpg",
+    ren_amina_ren : "images/ren amina.jpg",
+    het_diner : "images/het diner.jpg"
+    };
+ 
+  function bookList () { 
+
+    let bookcontainer = document.getElementById("mybooks");
+    let div = document.createElement("div");
+    bookcontainer.appendChild(div);
+    let li = document.createElement("li");
+    div.appendChild(li);
```

Comment:
I'm pretty sure you wanted this to be a `ul` tag.  It is the container for the rest of the list.